pages/kernels/index.js

```diff
@@ -6,5 +6,20 @@
  * yet we have to repeat ourselves sometimes. I think it's worth it. :)
  */
 
+import Head from "next/head";
+//Sections
+import Layout from "../../components/layout/layout";
+import ContentSection from "../../components/content-section/content-section";
+import PageHeader from './page-header'
+
 import Python from "./python";
-export default () => <Python />;
+
+
+export default () => (
+    <Layout pageTitle=": The nteract Desktop App">
+        <PageHeader />
+        <ContentSection>
+            <Python />
```

Comment:
Check out the changes in 3e65232e98e9eeff7ccabb37caf4b8bcf818f564. It primarily came down to extracting the base page setup into the `KernelPage` component so they inherited from the same base.